wfe/wfe.go

```diff
@@ -1882,6 +1891,18 @@ func (wfe *WebFrontEndImpl) FinalizeOrder(
 		return
 	}
 
+	// Ensure the signature on the CSR is good
+	if err := parsedCSR.CheckSignature(); err != nil {
+		wfe.sendError(acme.BadCSRProblem("Bad signature on CSR: "+err.Error()), response)
```

Comment:
Although I see it is done using concatenation on lines 1883 and 1890 above, we prefer string interpolation (as on line 1876, 1864, and 1857) in general:
```suggestion
		wfe.sendError(acme.BadCSRProblem(fmt.Sprintf("Bad signature on CSR: %s", err.Error())), response)
```